project1/application.py

```diff
@@ -16,40 +11,41 @@
     raise RuntimeError("DATABASE_URL is not set")
 
 # Configure session to use filesystem
-app.config["SESSION_PERMANENT"] = False
-app.config["SESSION_TYPE"] = "filesystem"
-Session(app)
-
-# Creating a database table USERS
-def create_register_table():
-    meta = MetaData()
-    USERS = Table(
-    'USERS', meta, 
-    Column('username', Integer, primary_key = True), 
-    Column('password', String))
-    meta.create_all(engine)
-    print("Database table created successfully")
-
-
-# Set up database
-engine = create_engine("postgres://tgsemzadyezlgh:858baa7b0d8b5ce3cb37d6481e187d508a119f33d2f26a2addac896f186eb633@ec2-18-233-137-77.compute-1.amazonaws.com:5432/d389gkjdhv6oa2", echo = True)
-db = scoped_session(sessionmaker(bind=engine))
-create_register_table()
+# app.config["SESSION_PERMANENT"] = False
+# app.config["SESSION_TYPE"] = "filesystem"
+# Session(app)
 
+app.config["SQLALCHEMY_DATABASE_URI"] = "postgres://tgsemzadyezlgh:858baa7b0d8b5ce3cb37d6481e187d508a119f33d2f26a2addac896f186eb633@ec2-18-233-137-77.compute-1.amazonaws.com:5432/d389gkjdhv6oa2"
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+db.init_app(app)
 
 @app.route("/")
 def index():
     return "Project 1: TODO"
 
-def show_registration_page():
-    return render_template('register.html')
+def register_user(request):
+    req = request.form
+    cnt_users = User.query.filter_by(username = req.get("username")).count()
```

Comment:
How about inserting the record without quering the database and catching an  exception to show the error message? It would save one connection to the database, right?

Reply:
Correct. I will do that

Reply:
Done